NEWS.md

```diff
@@ -1,5 +1,14 @@
 # usethis (development version)
 
+* `use_rstudio()` now sets the `LineEndingConversion` to your platform default.
```

Comment:
TBH I am not sure that the platform dependent default is right.

E.g. consider that X creates a project on Windows, so CRLF is the default. But X has line ending conversion in the git setup, so the repo will have LF. Then Y checks it out on Unix, line endings are still LF. Adds a new file, that will have CRLF line endings. Now the repo has a single file with CRLF. X checks it out again, changes something in the new file, commits, and git converts all file endings to LF, causing spurious changes.

It seems to be that simply having an LF default is better.

Also, doesn't `R CMD check` converts at least the code (and man?) files to LF as well?

Reply:
Yeah, I think you're right — now that we respect this field in usethis, we might as well say packages should use LF, in the same way that we say packages should use UTF-8.